src/Tickets/Commerce/Order_Modifiers/Models/Order_Modifier_Relationships.php

```diff
@@ -140,6 +125,6 @@ public static function fromQueryBuilderObject( $object ) {
 	 * @return array The object properties as an array.
 	 */
 	public function to_array(): array {
-		return $this->attributes;
+		return $this->toArray();
```

Comment:
Might as well mark this method as deprecated. Same for the `Order_Modifier` class in this directory. I think `Order_Modifier` is the only one that actually calls `->to_array()` somewhere, though I could be wrong. We can just update the call to use `->toArray()`.